dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -288,8 +293,6 @@ if [ "${CHE_MULTI_USER}" == "true" ]; then
     else
         "${COMMAND_DIR}"/multi-user/deploy_postgres_only.sh
     fi
-
-    "${COMMAND_DIR}"/multi-user/wait_until_postgres_is_available.sh
```

Comment:
I don't see why you remove this line. Allowing the Che server to start before either the Keycloak server or the che server leads to Che startup error, and the resuling pods do not work correctly.

afaik , by removing this line, you allow the Che server to start before the Postgres server is up and running, at least in the use-case when the dedicated Keycloak server is not used.

Reply:
I see what you mean. In my case check of Postgres was broken and was called twice in `deploy_postgres_only.sh` and here. So I removed second call, but now I see that it was needed for `deploy_postgres_only.sh`.
I'm going to add `"${COMMAND_DIR}"/multi-user/wait_until_postgres_is_available.sh` right after `"${COMMAND_DIR}"/multi-user/deploy_postgres_only.sh`
Are you OK with this solution?

Reply:
yes

Reply:
That would be great to also correctly support the case when an external Keycloak server is used.

Reply:
done

Reply:
Thanks.